vispy/gloo/wrappers.py

```diff
@@ -677,7 +676,12 @@ def get_gl_configuration():
     # XXX eventually maybe we can ask `gl` whether or not we can access these
     gl.check_error('pre-config check')
     config = dict()
-    gl.glBindFramebuffer(gl.GL_FRAMEBUFFER, 0)
+    canvas = get_current_canvas()
+    if canvas and hasattr(canvas, '_backend'):
```

Comment:
when does `canvas` not have `_backend`?

Reply:
I don't know. This was added by you.

Reply:
Maybe it is unnecessary now. If you feel up to removing them to see if it breaks anything go for it. If not, we can try removing it later if someone is motivated to

Reply:
I took out the `hasattr` check. I could see this having problems with some backends, but right now they all (including ipynb) are based on the `BaseCanvasBackend` which sets this `_backend`.